server/src/domain/storage/storage.core.ts

```diff
@@ -140,29 +143,74 @@ export class StorageCore {
 
     let move = await this.moveRepository.getByEntity(entityId, pathType);
     if (move) {
+      console.log('Found existing move record for', entityId, pathType, oldPath, newPath);
```

Comment:
You're probably removing removing this anyways, just wanting to make sure it doesn't get overlooked when finalizing the PR :)

Reply:
Yea, already have that removed locally actually :laughing: